src/cloud/docker/docker-syncing-data.md

```diff
@@ -1,17 +1,23 @@
 ---
 group: cloud-guide
-title: Synchronizing data in Docker
+title: Synchronizing data in a Docker developer environment
 functional_areas:
   - Cloud
   - Setup
   - Configuration
 ---
-In the Docker environment, the {{site.data.var.ee}} application works only if the Docker containers have access to the {{ site.data.var.ee }} application data. You can provide access  either by directly mapping the current working directory or by using a file synchronization tool.
 
-You must use a file synchronization tool if you are using [Docker Desktop](https://www.docker.com/products/docker-desktop) for Windows or macOS. This tool is required because the filesystem for the Docker containers is inside a Virtual Machine that the Docker Desktop is creating. The application files must be local to the containers.
+In a Docker development environment, the {{site.data.var.ee}} application works only if the Docker containers have access to the {{site.data.var.ee}} application data. You can provide access either by directly mapping the current working directory, or by using a file synchronization tool.
 
-## Native
-The Magento Cloud Docker uses the `/app/` directory inside the containers by default. If you are using Linux native, you can add the `--sync-engine="native` option to the `docker:build` command, which maps the current working directory to the `/app/` directory to eliminate the data synchronization requirement.
+The {{site.data.var.mcd}}  `docker-build` command provides the `--sync-engine <type>` option to select the file synchronization behavior when you build the `docker-compose.yml` configuration file. You can select from the following values:
+
+`--sync-engine` value | Description
+--------------------- | ------------
+`native` | Maps the current working directory to the `/app` directory on each volume, which provides direct access to the data without requiring any synchronization. The `native` option is the default and works well on Linux native hosts. On macOS or Windows hosts, this option results in extremely slow performance in the Docker environment.
+`mutagen` | Uses [Mutagen] for file synchronization. When you select Mutagen, you must [install Mutagen] on your host operating system before you [launch Docker in developer mode]. Use this option on macOS or Windows hosts.
+`docker-sync` | Uses [docker-sync] for file synchronization. When you select docker-sync, you must [install docker-sync] on your host operating system before you [launch Docker in developer mode]. Use this option on macOS or Windows hosts.
+
+Launching the Docker development environment using the native file synchronization option maps to the `/app` folder for the following containers:
```

Comment:
This sentence reads a little bizarre. Launching..maps to folder...I kind of understand it because  it was already stated in the table and there is a container list below, but the wording could be better. Here is an option:

```suggestion
The working directory maps to the volume `/app` directory for each of the following containers when you use the `native` file synchronization option to launch the Docker development environment:
```